Tests/L0/SonarQubePostTest/PRCA/ReportProcessorTests.ps1

```diff
@@ -0,0 +1,39 @@
+[CmdletBinding()]
+param()
+
+Write-Host "$PSScriptRoot\..\..\..\..\Tasks\Common\SonarQubeHelpers\SonarQubeHelper.ps1"
+Write-Verbose "$PSScriptRoot\..\..\..\..\Tasks\SonarQubePostTest\Common\SonarQubeHelpers.ps1"
+
+. $PSScriptRoot\..\..\..\lib\Initialize-Test.ps1
+. $PSScriptRoot\..\..\..\..\Tasks\SonarQubePostTest\Common\SonarQubeHelpers\SonarQubeHelper.ps1
+#. $PSScriptRoot\..\..\..\..\Tasks\Common\SonarQubeHelpers\SonarQubeHelper.ps1
+. $PSScriptRoot\..\..\..\..\Tasks\SonarQubePostTest\PRCA\Orchestrator.ps1
+
+#
+# This test validates the ReportProcessor that is responsible for loading the new issues 
+# from the SonarQube json report and computing the relative path to the associated code file. 
+#
```

Comment:
This test uses a sonar-project.json and some projectInfo.xml files on disk. Sadly the build process only copies json and ps1 files. As such I propose chaning the build process to copy over any "resources" directory. PR is https://github.com/Microsoft/vso-agent-tasks/pull/1316/files